.github/workflows/chromatic-dependabot.yml

```diff
@@ -0,0 +1,23 @@
+name: Dependabot Chromatic
+on:
+  workflow_run:
+    workflows: ['Dependabot PR Check']
+    types:
+      - completed
+
+permissions:
+  checks: write
```

Comment:
This permission is for the `GITHUB_TOKEN`, right? Does chromatic need write permissions or does it just need read to be able to checkout the branch?

Reply:
You're probably right. I'll remove this and see how it gets on without it.